ktor-http/ktor-http-cio/common/src/io/ktor/http/cio/HttpParser.kt

```diff
@@ -204,6 +210,7 @@ private fun parseStatusCode(text: CharSequence, range: MutableRange): Int {
     return status
 }
 
+private fun statusOutOfRange(code: Int) = code < HTTP_STATUS_CODE_MIN_RANGE || code > HTTP_STATUS_CODE_MAX_RANGE
```

Comment:
Status codes higher than 599 (e.g. 6xx) aren't allowed; however, some services may use it despite that. So I would relax this limitation and apply the [HTTP syntax limitation](https://tools.ietf.org/html/rfc7230#section-3.1.2) that is defined like this:
>      status-code    = 3DIGIT

Reply:
Thank you so much for your feedback! I'll fix it now.

Reply:
Just made a new commit to fix this limitation.